framework/architecture/businessunit/hierarchical-business-units.rst

```diff
@@ -0,0 +1,165 @@
+Hierarchical Business Units
+===========================
+
+Required Nice version: **2.17 and higher**
+
+Usage
+-----
+Hierarchical business units can be used to restrict read- and write-access for specified entity models in a hierarchical
+business unit structure. How the read/write access is restricted is defined by the chosen strategy. There are two
+different strategies available. `hierarchical` and `self`. It is possible to choose different strategies for the read
+access and the write access for a single entity model.
```

Comment:
"... for a single entity model" sounds to me like this can be done for only one entity model. Is that the case? If not, maybe write "... for every entity model"?

Reply:
done